script.js

```diff
@@ -1,15 +1,35 @@
-let hash = location.href.split("#")[1];
+const search = location.search.substring(1); // Gets the location.search, and then removes the first character
 
-if (hash) {
-    let image = document.createElement('img');
+(function() {
+    if (!search) return; // if the search empty, then it stops.
+
+    // image
+    let image = document.createElement("img");
+    image.src = `https://assets.scratch.mit.edu/get_image/../${search}`;
```

Comment:
What's the `/../` for?

Reply:
the original string was `https://assets.scratch.mit.edu/get_image/.%2E/${hash}`, `%2E` is the url encoding for `.`(period); so I just replaced it with `.` and it worked. Btw, for some reason `https://assets.scratch.mit.edu/get_image/${hash}` doesn't work

Reply:
ok thats fine